tests/forms_tests/tests/test_forms.py

```diff
@@ -3066,6 +3067,28 @@ class SomeForm(Form):
         self.assertHTMLEqual(form['custom'].label_tag(), '<label for="custom_id_custom">Custom:</label>')
         self.assertHTMLEqual(form['empty'].label_tag(), '<label>Empty:</label>')
 
+    def test_boundfield_label_tag_performance(self):
+        counter = 0
+
+        def my_field_label():
+            nonlocal counter
+            counter += 1
+            return 'a field label'
```

Comment:
Since lazyness is often about translations you could add the value of `get_language` here and run the test below twice, with different languages, to make it more obvious that labels still appear in the current language?